clamav-http/server/v0/ping_handler.go

```diff
@@ -0,0 +1,29 @@
+package v0
+
+import (
+	"net/http"
+
+	"github.com/dutchcoders/go-clamd"
+	"github.com/sirupsen/logrus"
+)
+
+type PingHandler struct {
+	Address string
+	Logger  *logrus.Logger
+}
+
+func (ph *PingHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	c := clamd.NewClamd(ph.Address)
+	err := c.Ping()
+	if err != nil {
+		ph.Logger.WithError(err).Infof("ping: not responding")
+		w.WriteHeader(http.StatusOK)
```

Comment:
This shouldn't be `StatusOK` if the ping check failed
```suggestion
		w.WriteHeader(http.StatusInternalServerError)
```

Reply:
I agree totally but the v0 api is a clone of clamav-rest which responds this way

Reply:
Right, but there's no `v1alpha` version of this handler to fix/update that

Reply:
Yeah I don't want one I think this endpoint is a waste of time and v1alpha/healthz makes more sense